web-client/src/presenter/actions/EditDocketRecord/completeDocketEntryQCAction.js

```diff
@@ -32,6 +32,7 @@ export const completeDocketEntryQCAction = async ({
     createdAt: entryMetadata.dateReceived,
     docketEntryId,
     docketNumber,
+    leadDocketNumber,
```

Comment:
Nit: this might could use a test

Reply:
No, that is not a typo

Reply:
Not sure test is needed here.  This is added into the entryMetadata so it can be passed into the completeDocketEntryQCInteractor where it is tested there.